app/styles/_feeds.scss

```diff
@@ -82,37 +52,46 @@
 		box-shadow: 0;
 		border-radius: 0;
 		text-align: center;
-		
 	}
-
 }
+
 .success {
-	/*left: 0;*/
-    position: absolute;
-    text-align: center;
-    width: 100%;
-    /*margin-top: 50px;*/
+  position: absolute;
+  text-align: center;
+  width: 100%;
 }
 
 
-.input-form {
+.input-form-wrapper {
```

Comment:
I renamed `input-form` as `input-form-wrapper`, I thought this name is bit more clear about what it is doing, it has border for fours sides.